Data-Monkey/netdata.xml

```diff
@@ -81,19 +83,28 @@ Live demo: https://learn.netdata.cloud/docs/live-demo&#xD;
   <Config Name="Claim Rooms" Target="NETDATA_CLAIM_ROOMS" Default="" Mode="" Description="Only needed if you want to use NetData Cloud.  Get this from https://app.netdata.cloud, Integrations, Docker &amp;amp; Kubernetes, Docker.  Then copy the value of NETDATA_CLAIM_ROOMS without the trailing \" Type="Variable" Display="always" Required="false" Mask="false"></Config>
   <Config Name="Extra Packages" Target="NETDATA_EXTRA_DEB_PACKAGES" Default="" Mode="" Description="Any extra packages that are needed (https://learn.netdata.cloud/docs/installing/docker#adding-extra-packages-at-runtime)" Type="Variable" Display="always" Required="false" Mask="false"/>
+  <Config Name="passwd" Target="/host/etc/passwd" Default="/etc/passwd" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/etc/passwd</Config>
+  <Config Name="group" Target="/host/etc/group" Default="/etc/group" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/etc/group</Config>
+  <Config Name="localtime" Target="/etc/localtime" Default="/etc/localtime" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/etc/localtime</Config>
   <Config Name="proc" Target="/host/proc" Default="/proc" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/proc</Config>
   <Config Name="sys" Target="/host/sys" Default="/host/sys" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/sys</Config>
-  <Config Name="doker.sock" Target="/var/run/docker.sock" Default="/var/run/docker.sock" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/var/run/docker.sock</Config>
   <Config Name="os-release" Target="/host/etc/os-release" Default="/etc/os-release" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/etc/os-release</Config>
-  <Config Name="passwd" Target="/host/etc/passwd" Default="/etc/passwd" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/etc/passwd</Config>
-  <Config Name="group" Target="/host/etc/group" Default="/etc/group" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/etc/group</Config>
+  <Config Name="log" Target="/host/var/log" Default="/var/log" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/var/log</Config>
+  <Config Name="doker.sock" Target="/var/run/docker.sock" Default="/var/run/docker.sock" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/var/run/docker.sock</Config>
+  <Config Name="dbus" Target="/run/dbus" Default="/run/dbus" Mode="ro" Description="" Type="Path" Display="advanced" Required="true" Mask="false">/run/dbus</Config>
```

Comment:
This mount is optional for systemd systems, but unRAID does make it available, so I mount it

Reply:
thanks for the PR